makefiles/simulators/Makefile.aldec

```diff
@@ -62,9 +62,19 @@ RTL_LIBRARY ?= work
 ALOG_ARGS += +define+COCOTB_SIM -dbg -pli libgpi
 ACOM_ARGS += -dbg
 
+# Aldec-specific coverage types:
+# - (s)tatement
+# - (b)ranch
+# - (e)xpression
+# - (c)ondition
+# - (a)ssertion
+# - (p)ath
+# - finite state (m)achine
+# Documentation: Riviera Pro 2017.02 Documentation - Page 359
+COVERAGE_TYPES ?= sb
 ifeq ($(COVERAGE),1)
-    ASIM_ARGS += -acdb
-    ALOG_ARGS += -coverage sb
+    ASIM_ARGS += -acdb -acdb_cov $(COVERAGE_TYPES)
+    ALOG_ARGS += -coverage $(COVERAGE_TYPES)
```

Comment:
As currently ```$COVERAGE``` variable is not supported in any other simulator makefile, I would address this topic separately. Personally, I don't think it should be the part of Makefile.* as it will be difficult to provide a consistent syntax accross all simulators. You can use ```EXTRA_ARGS``` in your project Makefile to add coverage options.